keps/sig-network/2829-gateway-api-to-k8s-io/README.md

```diff
@@ -0,0 +1,163 @@
+# KEP-2829: Migrate Gateway API to k8s.io Group
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+- [Proposal](#proposal)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Graduation Criteria](#graduation-criteria)
+    - [Alpha](#alpha)
+    - [Beta](#beta)
+- [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Feature Enablement and Rollback](#feature-enablement-and-rollback)
+  - [Rollout, Upgrade and Rollback Planning](#rollout-upgrade-and-rollback-planning)
+  - [Monitoring Requirements](#monitoring-requirements)
+  - [Dependencies](#dependencies)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+*Note: This KEP is primarily for tracking purposes. The code for this feature is
+all out-of-tree, but the using the k8s.io API group requires API review. Most
+sections of this KEP will be marked as not applicable.*
+
+This KEP aims to transition [Gateway API](https://gateway-api.sigs.k8s.io) from
+an experimental API (x-k8s.io) to an official one (k8s.io) as part of the
+v1alpha2 release. The APIs will continue to be represented as CRDs, but will be
+subject to the standard [Kubernetes API review
+process](https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md).
+
+## Motivation
+
+Although Gateway API started as an experimental API, we are preparing a v1alpha2
+release that we believe will be a significant step towards stabilizing the API.
+The number of [implementations of the
+API](https://gateway-api.sigs.k8s.io/references/implementations/) continues to
+grow, and we believe that transitioning to an official Kubernetes API will help
+provide greater quality and stability going forward.
+
+### Goals
+
+* Formalize transition of Gateway API from x-k8s.io API group to k8s.io.
+* Organize full Kubernetes API reviews as part of the transition.
+
+## Proposal
+
+This is a large API, it would not be practical to include it within this KEP.
+Instead, the API review process will happen through a separate PR or set of PRs
+on the [Gateway API repo](https://github.com/kubernetes-sigs/gateway-api).
+
+### Risks and Mitigations
+
+N/A
+
+## Design Details
+
+N/A
+
+### Graduation Criteria
+
+#### Alpha
+
+- Approval from subproject owners + KEP reviewers
+
+#### Beta
+
+- v1alpha2 APIs are implemented by several implementations
+- Approval from subproject owners + KEP reviewers
+- Initial conformance tests are in place
+- Validating webhook for advanced validation
```

Comment:
Reading https://kubernetes.io/docs/reference/using-api/, can we please add something along the lines "we know users of the API are using it in production and exercising most of the API surface"?
I'm assuming this list can be updated as we learn more, in which case, feel free to drop this as well.

Reply:
Updated, thanks!